src/Maestro/Maestro.Web/Maestro.Web.csproj

```diff
@@ -22,6 +22,8 @@
     <PackageReference Include="JetBrains.Annotations" /> 
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.AzureKeyVault" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection.Azurestorage" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" VersionOverride="2.1.22" />
```

Comment:
??? Why a version override?  If we want to change the version, we should always be doing it in Packages.props unless a particular project REALLY needs a different version.

Reply:
Love the idea of a comment, will add. 

Using VersionOverride was just my first instinct. I'm happy to move it to Packages.props.

Reply:
Yup.  We just need to add this assembly to the packages.props to lock it's version. Presumably right now the version is just floating to whatever the SDK picks.

Reply:
It'll still need an explicit reference (just without the VersionOverride) in each project, yes?

Reply:
Yup. The "PackageReference" needs to stay, i think. But the "Version" can still be handled in the shared place, so we can update it easier next time.  I like that we are sloooowly just having to add every framework assembly here because of the way they get included.